.github/workflows/continuous-integration.yml

```diff
@@ -485,8 +561,8 @@ jobs:
   testing-reports:
     name: Testing Reports
     runs-on: ubuntu-latest
-    needs: cypress-tests
-    if: ${{ always() && (needs.cypress-tests.result == 'success' || needs.cypress-tests.result == 'failure') }}
+    needs: [cypress-tests-prep, cypress-tests]
+    if: ${{ always() && (github.ref == 'refs/heads/master' || needs.cypress-tests-prep.outputs.num_containers > 0) && (needs.cypress-tests.result == 'success' || needs.cypress-tests.result == 'failure') }}
```

Comment:
Can this be changed to `if: ${{ github.ref == 'refs/heads/master' }}`? Do we need to generate the report for non-master builds?

Reply:
I spoke to Peter about this and we want to push all test results to BigQuery, even if only a subset of tests run due to test selection. So we need `always()`.